packages/manager/apps/dedicated/client/app/ip/components/list/byoip/aggregate/ip-ip-byoip-aggregate.html

```diff
@@ -0,0 +1,37 @@
+<div data-ng-controller="IpAggregateByoipCtrl as $ctrl">
+    <div
+        data-wizard
+        data-wizard-on-cancel="$ctrl.cancelAggregate.bind($ctrl)"
+        data-wizard-on-finish="$ctrl.aggregate.bind($ctrl)"
+        data-wizard-title="('ip_byoip_aggregate_block_title' | translate) + ' ' + $ctrl.ip.ipBlock"
+    >
+        <div data-wizard-step data-wizard-step-valid="$ctrl.isLoaded">
+            <div data-ng-show="$ctrl.isLoaded">
+                <oui-message data-type="error" data-ng-show="$ctrl.errorMessage" >
+                    <span data-ng-bind="$ctrl.errorMessage"></span>
+                </oui-message>
+                <p data-translate="ip_byoip_aggregate_set_parent_size"></p>
+                <label class="oui-select">
+                    <select data-ng-model="$ctrl.selectedAggregationIp" class="oui-select__input" >
+                        <option data-ng-repeat="aggregationIp in $ctrl.aggregationIps track by $index" data-ng-value="aggregationIp">
+                            <span>{{aggregationIp.aggregationIp}}</span>
```

Comment:
```suggestion
                            <span data-ng-bind="aggregationIp.aggregationIp"></span>
```

Reply:
I didn't see this was inside an option tag. the data-ng-bind property should be on the option tag itself and you can remove the span inside

Reply:
done